build.cmd

```diff
@@ -197,6 +198,26 @@ exit /b 1
 
 REM Build CoreCLR
 :BuildCoreCLR
+:: Run Steps to Generate ETW specific infrastructure the 
+if not exist "%__GeneratedIntermediatesDir%" md "%__GeneratedIntermediatesDir%"
```

Comment:
Do you want to delete this folder if it exists to delete any previously created files?